Tweeter.Tests/DAL/TweeterRepoTests.cs

```diff
@@ -62,7 +74,25 @@ public void RepoEnsureCanCreateInstance()
         [TestMethod]
         public void RepoEnsureICanGetUsernames()
         {
+            var repo = new TweeterRepository(mock_context.Object);
+
+            var allUsernames = repo.GetUsernames();
+
+            Assert.AreEqual(allUsernames.Count, 1);
+        }
+
+        [TestMethod]
+        public void RepoEnsureICanGetByUsernames()
+        {
+            var repo = new TweeterRepository(mock_context.Object);
+
+            var newUser = new ApplicationUser() { UserName = "Superman" };
+
+            users.Add(newUser);
+
+            var foundUser = repo.GetUsernames("Superman");
```

Comment:
I don't like the name of the method here -- make sure your method names are descriptive. Does this really GetUsernames? or is it a GetUserByUserName?